test/RoaveTest/SecurityAdvisories/VersionTest.php

```diff
@@ -208,8 +167,8 @@ public function greaterOrEqualThanComparisonVersionsProvider() : array
             ['1.2', '1.100', false, true],
             ['1.1', '1.1.0', true, true],
             ['1.1', '1.1.0.0', true, true],
-            ['1.1', '1.1.0.0.1', false, true],
-            ['1.0.0.0.0.0.2', '1.0.0.0.0.2', false, true],
```

Comment:
Same as above: shouldn't be a change, but rather an addition to the existing test